test/client/filters.clienttest.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  'Filtering on search page': function (browser) {
+    browser
+      .url('http://localhost:8000/search?q=charles')
```

Comment:
this Nightwatch test looks *good*. 
are you running it from CI?
you don't _appear_ to be saving any screenshots of the output, might be a good addition. 
did you see: https://github.com/dwyl/learn-nightwatch/pull/27 ?